cockroachcloud/managing-cmek.md

```diff
@@ -0,0 +1,125 @@
+
+	They will enable the [Cloud API](cloud-api.html) and CMEK for your {{ site.data.products.dedicated }} Organization.
+
+	{{site.data.alerts.callout_info}}
+	The upgrade process will generate a new Organization ID. You will need this new Organization ID when provisioning the crucial cross-account IAM role that will allow Cockroach Labs to encrypt/decrypt on your behalf with the CMEK.
+	{{site.data.alerts.end}}
+
+1. [Create a {{ site.data.products.db }} service account](console-access-management.html#service-accounts).
+
+1. [Create an API key](console-access-management.html#create-api-keys) for the service account to use.
+
+### Step 2. Provision your cluster
+
+1. Create a new {{ site.data.products.dedicated }} cluster. You can do this either of two ways:
+	- [Using the {{ site.data.products.db }} console clusters page](https://cockroachlabs.cloud/cluster).
+	- [Using the Cloud API](cloud-api.html#create-a-new-cluster). 
+
+### Step 3. Provision IAM and KMS in your IaaS
+
+Next, you must provision the resources required resources in your IaaS, whether this is AWS or GCP:
+
+1. The cross-account IAM role that Cockroach Labs will use to access your key.
+
+1. The key itself.
+
+Follow the instructions depending on your IaaS:
+
+- [Provisioning Amazon Web Services (AWS) for CMEK](cmek-ops-aws.html)
+- [Provisioning Google Cloud Platform (GCP) for CMEK](cmek-ops-gcp.html) 
+
+### Step 4. Enable CMEK for your CockroachDB Cluster
+
+## Check CMEK status
+
+An API call displays information about your cluster's use of CMEK: 
+
+[API specification](../api/cloud/v1.html#operation/CockroachCloud_GetCMEKClusterInfo)
+
+{% include_cached copy-clipboard.html %}
+```shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/cmek \
+  --header 'Authorization: Bearer REPLACE_BEARER_TOKEN'
+```
+
+## Revoking CMEK for a cluster
+
+Revoking access to the CMEK means disabling all encryption/decryption of data in your database, which means preventing reading and writing any data from or to your database. This likely implies a shutdown of your service, with significant business implications. This is a disaster mitigation tactic to be used only in scenario involving a severe, business critical security breach.
+
+This can be done temporarily or permanently. This action is performed at the level of your infrastructure (IaaS).
+
+### Step 1. Revoke IAM access
+
+Revoke CMEK for a cluster by revoking {{ site.data.products.db }}'s access to your key at the IAM level with your infrastructure provider. This will immediately prevent all access to your data, but can be reversed by reauthorizing access to the key.
+
+Deleting the key will permanently prevent decryption of your data, preventing all possible access and rendering the database unusable and data inaccessible.
+
+### Step 2. Assess and resolve the situation
+
+Eventually, you'll resolve the security incident and re-authorize CMEK for your cluster to return to normal operations.
```

Comment:
Maybe here: 

"Once you have resolved the security incident, re-authorize CMEK for your cluster..."

To remove the future tense and "eventually", which sounds a little off – like this is going to take some time (even if it is).